packages/experimental/Switch/src/__tests__/__snapshots__/Switch.test.tsx.snap

```diff
@@ -35,20 +35,18 @@ exports[`Switch Default 1`] = `
       "alignItems": "center",
       "alignSelf": "flex-start",
       "borderColor": undefined,
-      "borderRadius": 4,
-      "borderWidth": 2,
+      "borderRadius": undefined,
```

Comment:
Why did this snapshot change? Is it because it's testing Android or something else?

Reply:
Yes , Actually the jest test runs for Android as provided on the jest.config here : 

https://github.com/microsoft/fluentui-react-native/blob/main/packages/experimental/Switch/jest.config.js 

Since there was no android specific tokens it was taking the default one , now based on design few tokens needed removal of it , hence there is an update on the snapshots.